driver-adapters-wasm/planetscale-vercel-nextjs-edgemw/prisma/schema.prisma

```diff
@@ -6,8 +6,8 @@ generator client {
 
 datasource db {
   provider = "mysql"
-  // TODO: needs to be changed to a fake url to ensure the driver adapter is used
-  url      = env("DRIVER_ADAPTERS_PLANETSCALE_VERCEL_NEXTJS_EDGEMW_DATABASE_URL")
+  // TODO this url should not contain the db name
+  url      = "mysql://foo:bar@baz/eco-driver-adapters-planetscale-vercel-nextjs-edgemw?sslaccept=strict"
```

Comment:
Not sure what these TODO are for

Reply:
We have an issue here https://github.com/prisma/team-orm/issues/662